tests/test_view/core/user_login_test.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+
+
+@pytest.mark.django_db
+def test_user_login(get_auth_client, user_factory):
+    """
+    Test user auth
+    """
+    user = user_factory()
```

Comment:
Не вижу смысла создавать пользователя с паролем и менять пароль, можно сразу задать нужный пароль
```suggestion
    user = user_factory(password='123456')
```